install-php-extensions

```diff
@@ -540,8 +543,16 @@ buildRequiredPackageLists () {
 				buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile cmake gnutls-dev libzip-dev libressl-dev zlib-dev"
 				;;
 			zip@debian)
-				buildRequiredPackageLists_persistent="$buildRequiredPackageLists_persistent libzip4 libmbedtls1?"
-				buildRequiredPackageLists_volatile="$buildRequiredPackageLists_volatile cmake gnutls-dev $buildRequiredPackageLists_libssldev libzip-dev libbz2-dev libmbedtls-dev zlib1g-dev"
+				case "$(getDistroVersion)" in
+					debian@8)
+						buildRequiredPackageLists_persistent="$buildRequiredPackageLists_persistent libzip2"
```

Comment:
we can write `libzip[0-9]`, so that we don't have to handle different package names for different Debian versions